tests/installation/installation_overview.pm

```diff
@@ -42,6 +42,9 @@ sub ensure_ssh_unblocked {
                 send_key_until_needlematch 'ssh-blocked-selected', 'tab', 25;
                 send_key 'ret';
                 send_key_until_needlematch 'ssh-open', 'tab';
+		send_key_until_needlematch 'ssh-service-enabled', 'tab';
+		send_key_until_needlematch 'confirm-installation', 'tab';
```

Comment:
The confirmation for the installation happens later, outside of `ensure_ssh_unlocked`.

The problem that happened in the verification run is that the return key was sent when the button "install" was focused (instead of the sshd option).